linetools/lists/sets/llist_v1.1.ascii

```diff
@@ -1121,7 +1121,7 @@
 | 1200.7098 |    NI 1200c |    1 |   1 |   0 |   0 |   0 |    0 |   0 |   0 |    0 |
```

Comment:
It is a bit odd to add these lines to the EUV list, i.e. they
aren't EUV lines.  I'd say it is better to read them in via
the ISM or Strong sets.